setup.cfg

```diff
@@ -30,6 +30,7 @@ filterwarnings =
 ;    Unfortunately due to https://github.com/pytest-dev/pytest/issues/8343 we can't have this here
 ;    and instead do a trick directly in tests/conftest.py
 ;    ignore::telegram.utils.deprecate.TelegramDeprecationWarning
+markers = dev: If you want to test a specific test, use this
```

Comment:
please remove

Reply:
But I like it for development.

Reply:
we've added it to v20 for that reason. v13 is supposed to get api updates exclusively, so I'd like to stick to that as close as reasonbly possible :)

Reply:
It's such a small non breaking change though. And it just really eases testing specific tests.

Reply:
fine

Reply:
Yes! xD